tests/pytests/common.py

```diff
@@ -354,7 +354,7 @@ def wrapper(env, *args, **kwargs):
 def skipTest(**kwargs):
     skip(**kwargs)(lambda: None)()
 
-def skip(cluster=None, macos=False, asan=False, msan=False, noWorkers=False, redis_less_than=None, redis_greater_equal=None, min_shards=None, arch=None, gc_no_fork=None):
+def skip(cluster=None, macos=False, asan=False, msan=False, noWorkers=False, redis_less_than=None, redis_greater_equal=None, min_shards=None, arch=None, gc_no_fork=None, **kwargs):
```

Comment:
wasn't that removed from other branches?

Reply:
looks like. Will revert